### DISCUSSION TITLE
"dataset was not found in location US" error in get_historical_features()

### DISCUSSION THREAD

**Discussion by Jinwon-Lee** [feast-dev/feast#1808] [Q&A]
I got the following error message .
NotFound: 404 Not found: Dataset {project}:{dataset} was not found in location US

When connecting to bigquery from GCP, is it possible only in the US region?
Is there any way to configure yaml or something without modifying common modules?

**Comment by francoisserra** [comment]
Hi, for me your problem is probably fixed by this PR/bugfix [Create the BQ dataset in the location specified in feature_store.yaml if the dataset doesn't exists](https://github.com/feast-dev/feast/pull/1959)
